internal/recert/recert.go

```diff
@@ -69,6 +69,9 @@ type RecertConfig struct {
 	PullSecret                string   `json:"pull_secret,omitempty"`
 	ChronyConfig              string   `json:"chrony_config,omitempty"`
 	RegenerateServerSSHKeys   string   `json:"regenerate_server_ssh_keys,omitempty"`
+
+	// defragment etcd after recertification
+	EtcdDefrag bool `json:"etcd_defrag,omitempty"`
```

Comment:
Don't think documenting recert options is necessary, recert has documentation already